pallets/capacity/src/lib.rs

```diff
@@ -198,10 +198,10 @@ pub mod pallet {
 		/// Note that you can claim rewards even if you no longer are boosting, because you
 		/// may claim rewards for past eras up to the history limit.
 		#[pallet::constant]
-		type StakingRewardsPastErasMax: Get<u32>;
+		type ProviderBoostRewardsPastErasMax: Get<u32>;
 
-		/// The StakingRewardsProvider used by this pallet in a given runtime
-		type RewardsProvider: StakingRewardsProvider<Self>;
+		/// The BoostingRewardsProvider used by this pallet in a given runtime
```

Comment:
I called it "BoostingRewardsProvider" because "ProviderBoostRewardsProvider" seemed a bit much. Welcoming alternatives.

Reply:
I prefer ProviderBoostRewardsProvider it reads better.